docs/html-css/week-1/lesson.md

```diff
@@ -70,42 +117,140 @@ As a group, let's try to name all of the parent and child tags in the following
 
 ## Example HTML/CSS Project
 
-In today's class, we will begin adapting styles on this example website. We'll review some of the HTML/CSS basics you already encountered during your application process and learn some new techniques. By the end of the third lesson, we will have worked together to improve the example site on the left so that it looks like the screenshot on the right.
+In today's class, we will begin adapting styles on this example website. We'll review some of the HTML/CSS basics you already encountered during your application process and learn some new techniques. By the end of the third lesson, we will have worked together to improve the example site on the left screenshot below so that it looks like the screenshot below on the right.
 
 <a href="../assets/screenshot-start.png" target="blank">
- <img src={require('!file-loader!../assets/screenshot-start.png').default}/>
+ <img src={require('!file-loader!../assets/screenshot-start.png').default} alt="Bikes for Refugees home page, unstyled" width="50%"/>
 </a>
 <a href="../assets/screenshot-complete.png" target="blank">
- <img src={require('!file-loader!../assets/screenshot-complete.png').default}/>
+ <img src={require('!file-loader!../assets/screenshot-complete.png').default} alt="Bikes for Refugees home page, styled" width="50%"/>
 </a>
 
-The example website you'll begin working with is available on this Code Your Future GitHub repository -
-[Bikes for Refugees](https://github.com/CodeYourFuture/bikes-for-refugees). Fork the repository to your personal account and then clone the repository
+The example website you'll begin working with is available on the Code Your Future [Bikes for Refugees repository](https://github.com/CodeYourFuture/bikes-for-refugees) on GitHub.
+Fork the repository to your personal account and then clone the repository.
 
 ### Exercise (5 minutes)
 
 Spend a few minutes exploring the `.html` and `.css` files for this page. Why don't we put everything in one file?
 
 ## Semantic HTML
 
-When writing HTML code, you can use different tags to describe the content. Is it a navigation menu, a paragraph of text, or an article? By using the correct tag, you help search engines like Google or screen readers for the visually impaired.
+When writing HTML code, we can use different tags to describe the content. Is it a navigation menu, a paragraph of text, or an article? By using the correct tag, we help search engines like Google or screen readers for the visually impaired understand the content better.
+
+When possible, we should use semantic HTML instead of generic HTML like `<div>` and `<span>`.
 
 > Semantic HTML is the use of HTML markup to reinforce the semantics, or **meaning**, of the information in webpages and web applications rather than merely to define its presentation or look. [Wikipedia](https://en.wikipedia.org/wiki/Semantic_HTML)
 
-We'll cover the following semantic tags:
+### Sectioning content
```

Comment:
This and the next few sections expose students to some common HTML tags for familiarity and reference.

Reply:
Good idea. I've added a couple of extra exercises to get students to use the HTML elements in the new sections. Do they seem okay?